editor/src/editor/layout/inspector/material/components/transparency.tsx

```diff
@@ -0,0 +1,36 @@
+import { ReactNode } from "react";
+import { StandardMaterial, PBRMaterial } from "babylonjs";
+import { EditorInspectorListField } from "../../fields/list";
+
+export interface IEditorTransparencyModeFieldProps {
+	object: StandardMaterial | PBRMaterial;
+	onChange?: () => void;
+}
+
+export function EditorTransparencyModeField(props: IEditorTransparencyModeFieldProps): ReactNode {
+	const getTransparencyModeItems = () => {
+		const material = props.object;
+
+		if (material instanceof PBRMaterial) {
```

Comment:
I see what you did.
You don't need to separate PBRMaterial and StandardMaterial, you can use Material.X directly. It's available for all materials, like:

`
import { Material } from "babylonjs";

{ text: "None", value: null },
{ text: "Opaque", value: Material.MATERIAL_OPAQUE },
{ text: "Alpha Test", value: Material.MATERIAL_ALPHATEST },
{ text: "Alpha Blend", value: Material.MATERIAL_ALPHABLEND },
{ text: "Alpha Test and Blend", value: Material.MATERIAL_ALPHATESTANDBLEND },
`